packages/expo-cli/src/commands/export.ts

```diff
@@ -230,5 +234,6 @@ export default function (program: Command) {
       []
     )
     .option('--max-workers [num]', 'Maximum number of tasks to allow Metro to spawn.')
+    .option('--eas', 'export bundles for use with EAS updates.')
```

Comment:
If I'm not mistaken this just exports bundles from Metro, has nothing to do with EAS.